### ISSUE TITLE
Why student use log_softmax(), while teacher use softmax() ?

### ISSUE THREAD

**Issue by Ou-Rui** [peterliht/knowledge-distillation-pytorch#41]
![图片](https://user-images.githubusercontent.com/56575023/155940045-30e66b19-efa6-40ed-9a4f-fa49b4f554ef.png)
